<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Milad Library Injector!</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/new/prefix1/images/wolfIcon.jpg</normaloff>:/new/prefix1/images/wolfIcon.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QWidget" name="centralwidgetMainWindow">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelIcon">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/new/prefix1/images/wolfIcon.jpg</pixmap>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacerLeftSide">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayoutMain">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutLables">
          <item>
           <widget class="QLabel" name="labelLibrary">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Library Path:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelProcessID">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Process ID:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutLineEdit">
          <item>
           <widget class="QLineEdit" name="lineEditLibrary"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditProcessID"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutButtons">
          <item>
           <widget class="QPushButton" name="pushButtonLibrary">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgb(0, 85, 127);
	color:white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 7em;
    padding: 3px;
}

QPushButton:hover{
	background-color: rgb(184, 184, 184);
};</string>
            </property>
            <property name="text">
             <string>Select</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonProcessIdList">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgb(0, 85, 127);
	color:white;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 7em;
    padding: 3px;
}

QPushButton:hover{
	background-color: rgb(184, 184, 184);
};</string>
            </property>
            <property name="text">
             <string>List</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacerUp">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>375</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxOStType">
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:green;</string>
        </property>
        <property name="title">
         <string>Library Type:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayoutOSType">
           <item>
            <widget class="QRadioButton" name="radioButtonWindows">
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:blue</string>
             </property>
             <property name="text">
              <string>Windows</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonLinux">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:red</string>
             </property>
             <property name="text">
              <string>Linux</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButtonMacOS">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="font">
              <font>
               <family>Consolas</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color:brown</string>
             </property>
             <property name="text">
              <string>macOS</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutInfo">
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutLabelAddress">
          <item>
           <widget class="QLabel" name="labelDllAllocation">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Dll Alloc Address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelLoadLibrary">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>LoadLibrary Address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRemoteThread">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Remote Thread Address:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayoutAddress">
          <item>
           <widget class="QLabel" name="labelDllAllocationAddress">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:red;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelLoadLibraryAddress">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:green;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelRemoteThreadAddress">
            <property name="font">
             <font>
              <family>Consolas</family>
              <pointsize>9</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:blue;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacerDown">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>375</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutActionButton">
        <item>
         <widget class="QPushButton" name="pushButtonButtonInject">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	background-color: rgb(170, 85, 255);
	display:inline-block;
	padding:0.5em 1.5em;
	margin:0 0.3em 0.3em 0;
	border-radius:0.2em;
	box-sizing: border-box;
	text-decoration:none;
	font-family:'Consolas',sans-serif;
	font-weight:bold;
	color:#FFFFFF;
	box-shadow:inset 0 -0.6em 1em -0.35em rgba(0,0,0,0.17),inset 0 0.6em 2em -0.3em rgba(255,255,255,0.15),inset 0 0 0em 0.05em rgba(255,255,255,0.12);
	text-align:center;
	position:relative;
	height: 30px;
	width: 80px;
}

QPushButton:hover{
	background-color: rgb(184, 184, 184);
};

active{
	box-shadow:inset 0 0.6em 2em -0.3em rgba(0,0,0,0.15),inset 0 0 0em 0.05em rgba(255,255,255,0.12);
}

@media all and (max-width:30em){
 display:block;
 margin:0.2em auto;
}</string>
          </property>
          <property name="text">
           <string>Inject!</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/prefix1/images/select.png</normaloff>:/new/prefix1/images/select.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonButtonClose">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
	background-color: rgb(255, 0, 0);
	display:inline-block;
	padding:0.5em 1.5em;
	margin:0 0.3em 0.3em 0;
	border-radius:0.2em;
	box-sizing: border-box;
	text-decoration:none;
	font-family:'Consolas',sans-serif;
	font-weight:bold;
	color:#FFFFFF;
	box-shadow:inset 0 -0.6em 1em -0.35em rgba(0,0,0,0.17),inset 0 0.6em 2em -0.3em rgba(255,255,255,0.15),inset 0 0 0em 0.05em rgba(255,255,255,0.12);
	text-align:center;
	position:relative;
	height: 30px;
	width: 80px;
}

QPushButton:hover{
	background-color: rgb(184, 184, 184);
};

active{
	box-shadow:inset 0 0.6em 2em -0.3em rgba(0,0,0,0.15),inset 0 0 0em 0.05em rgba(255,255,255,0.12);
}

@media all and (max-width:30em){
 display:block;
 margin:0.2em auto;
}</string>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/new/prefix1/images/quit.png</normaloff>:/new/prefix1/images/quit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionProcessList"/>
    <addaction name="actionInjection"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutMLI"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAboutMLI">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/images/about.png</normaloff>:/new/prefix1/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About MLI</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/images/quit.png</normaloff>:/new/prefix1/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionProcessList">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/images/engineering.png</normaloff>:/new/prefix1/images/engineering.png</iconset>
   </property>
   <property name="text">
    <string>Process List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionInjection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/images/select.png</normaloff>:/new/prefix1/images/select.png</iconset>
   </property>
   <property name="text">
    <string>Injection</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
